webui/react/src/App.module.scss

```diff
@@ -6,4 +6,11 @@
   width: 100%;
 
   & > *:last-child { flex-grow: 1; }
```

Comment:
chore: `& > *:last-child` definition is no longer needed since `.body` has replaced it essentially